docs/conversion_examples_gallery/behavior/audio.rst

```diff
@@ -0,0 +1,30 @@
+Audio data conversion
+---------------------
+
+Install NeuroConv with the additional dependencies necessary for reading audio data.
+
+.. code-block:: bash
+
+    pip install neuroconv[audio]
+
+This interface can handle conversions from `WAV` format to NWB using the
+:py:class:`~neuroconv.datainterfaces.behavior.audio.audiointerface.AudioInterface` class.
+
+.. code-block:: python
+
+    >>> from datetime import datetime
+    >>> from dateutil import tz
+    >>> from pathlib import Path
+    >>>
+    >>> from neuroconv.datainterfaces.behavior.audio.audiointerface import AudioInterface
+    >>>
+    >>> audio_file_path = BEHAVIOR_DATA_PATH / "audio_recording.wav"
```

Comment:
Adding a new test file to https://gin.g-node.org/CatalystNeuro/behavior_testing_data/ is needed for testing the docs with this new tutorial.

Reply:
Thank you! I opened https://gin.g-node.org/CatalystNeuro/behavior_testing_data/pulls/4